### ISSUE TITLE
common prerequisites documentation should be in its own document

### ISSUE THREAD

**Issue by Xiaoy312** [unoplatform/uno#5795]
> I think we should redirect to the common prerequisites documentation, it may get outdated quickly.

_Originally posted by @jeromelaban in https://github.com/unoplatform/uno/pull/5632#discussion_r611273407_

**Comment by jeromelaban** [comment]
@davidjohnoliver we already have this in the documentation, right ?

**Comment by davidjohnoliver** [comment]
Yes, it should redirect to this page: https://github.com/unoplatform/uno/blob/master/doc/articles/get-started.md

**Comment by agneszitte** [comment]
@jeromelaban / @MartinZikmund  / @Xiaoy312
Is this issue still valid with the latest doc updates ?

**Comment by jeromelaban** [comment]
It is, yes, though I don't think we're using this template at this time. It's likely something that we'll need to put in our "docs on docs".
